sonic_api.yaml

```diff
@@ -109,6 +109,7 @@ info:
 schemes:
   - http
   - https
+host: localhost:9001
```

Comment:
Why port 9001? What is the significance?

Reply:
a host field was required for autorest, this is the smallest of the changes so I can just do this locally when I generate. I just used a random port (although what port is this using?)